033_j_L47.py

```
@@ -0,0 +1,80 @@
+        if file is not None and localhost is not None and not printed:
+            print('File:', file)
+            localhost = "http://" + localhost.split("http://")[1]
+            print('Localhost:', localhost)
+            printed = True
+            # start jupyter
+            jupyter = localcommand.format(host=host, file=file)
+            print(jupyter)
+            # {fact rule=os-command-injection@v1.0 defects=1}
+            os.system(jupyter)
+            # {/fact}
+    rc = process.poll()
```

Ruff (0.14.5)

Local variable `rc` is assigned to but never used

Remove assignment to unused variable `rc`

(F841)